### ISSUE TITLE
Transform deploy.sh to a python script using argcmdr and boto3

### ISSUE THREAD

**Issue by nanounanue** [dssg/dirtyduck#23]

**Comment by nanounanue** [comment]
Any pointers @jesteria ?

**Comment by jesteria** [comment]
@nanounanue Yep and [I wrote them all up just for you](https://github.com/dssg/argcmdr#argcmdr) 😉 

(If needed, there are examples in projects like [dssg/marketplace](https://github.com/dssg/marketplace) and [dssg/appy-reviews](https://github.com/dssg/appy-reviews/). As it happens, these generally execute _subprocesses_, and use `aws-cli`. This may or may not make sense for you.)

**Comment by nanounanue** [comment]
Wow! I will incorporate those! As always you are amazing @jesteria
